.github/workflows/pytest.yml

```diff
@@ -2,6 +2,10 @@ name: tests
 
 on: [push]
 
+concurrency:
```

Comment:
This option will only allow the latest CI run to run, cancelling any in progress jobs. This also speeds up iteration.